src/rqt_publisher/publisher_widget.py

```diff
@@ -56,13 +57,14 @@ class PublisherWidget(QWidget):
     remove_publisher = Signal(int)
     clean_up_publishers = Signal()
 
-    def __init__(self, parent=None):
+    def __init__(self, node, parent=None):
```

Comment:
take in context instead of passing around node

Reply:
Fixed